packages/vscode-extension/src/webview/components/Preview.tsx

```diff
@@ -116,8 +116,19 @@ type Props = {
 };
 
 function Preview({ isInspecting, setIsInspecting, zoomLevel, onZoomChanged }: Props) {
+  interface TouchPoint {
+    x: number;
+    y: number;
+  }
+
   const wrapperDivRef = useRef<HTMLDivElement>(null);
   const [isPressing, setIsPressing] = useState(false);
+  const [isMultiTouching, setIsMultiTouching] = useState(false);
+  const [isPanning, setIsPanning] = useState(false);
+  const [touchPoint, setTouchPoint] = useState<TouchPoint>({ x: 0.5, y: 0.5 });
+  const [mirroredTouchPoint, setMirroredTouchPoint] = useState<TouchPoint>({ x: 0.5, y: 0.5 });
```

Comment:
Can we always compute it instead of keeping it in the state?

Reply:
`mirroredTouchPoint` can be replaced with computing. I've changed it.
However, `moveAnchorPoint` needs to track previous mouse position so `touchPoint` state is used there before computing a new one. Also `getMirroredTouchPosition` function is called in place where mouse event is not available (after change it's used in TouchPointMarker component) so we need to store mouse position in `touchPoint`.

Reply:
`mirroredTouchPoint` can be replaced with computing. I've changed it.
However, `moveAnchorPoint` needs to track previous mouse position so `touchPoint` state is used there before computing a new one. Also `getMirroredTouchPosition` function is called in place where mouse event is not available (after change it's used in TouchPointMarker component) so we need to store mouse position in `touchPoint`.